### PR TITLE
Refactor

### LINKED ISSUE #12: Too many instances of logger
```
(node) warning: possible EventEmitter memory leak detected. 11 uncaughtException listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at process.addListener (events.js:239:17)
    at Logger.handleExceptions (/Users/fran/Documents/Development/Modulus/chap/node_modules/winston/lib/winston/logger.js:434:13)
    at Logger.add (/Users/fran/Documents/Development/Modulus/chap/node_modules/winston/lib/winston/logger.js:500:10)
    at /Users/fran/Documents/Development/Modulus/chap/node_modules/winston/lib/winston/logger.js:59:12
    at Array.forEach (native)
    at Logger.configure (/Users/fran/Documents/Development/Modulus/chap/node_modules/winston/lib/winston/logger.js:58:24)
    at new exports.Logger (/Users/fran/Documents/Development/Modulus/chap/node_modules/winston/lib/winston/logger.js:27:8)
    at module.exports (/Users/fran/Documents/Development/Modulus/chap/node_modules/@modulus/logger/lib/factory.js:6:10)
    at module.exports (/Users/fran/Documents/Development/Modulus/chap/node_modules/@modulus/logger/index.js:20:10)
    at Object.<anonymous> (/Users/fran/Documents/Development/Modulus/chap/index.js:10:10)
```

### PR DESCRIPTION
**Pull request by jackboberg**
This started as an attempt to solve the issue where every logger attached more EventEmiter listeners, leading to memory leaks:

`(node:15) Warning: Possible EventEmitter memory leak detected. 11 uncaughtException listeners added. Use emitter.setMaxListeners() to increase limit`

..in the end I accidentally the whole module. 👍 

Closes #12

---
_note: this removes the [pre-exit](https://github.com/XervoIO/logger#pre-exit-procedure) functionality..mostly a pointless idea IMO we never used it_

### CODE REVIEW

**Comment by jackboberg** [comment]
- [x] I'll need to update the `README` around exception logging, which is now transparent to the user and only requires a `EXCEPTION_LOG` environment variable (that is a writeable path).

**Comment by theworkflow** [comment]
@jackboberg would love to go over this PR.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Modulus logger
```
Comment (bsatrom) [inline README.md]: Modulus --> Xervo?

Comment (theworkflow) [inline README.md]: update name

**Review by bsatrom — APPROVED** [review]

**Review by theworkflow — CHANGES_REQUESTED** [review]

**Comment by jackboberg** [comment]
fixed README name

**Review by theworkflow — APPROVED** [review]

**Pull request merged**